docs/source/release/v6.12.0/Diffraction/Powder/New_features/38535.rst

```diff
@@ -0,0 +1 @@
+-  Now added an option for the vanadium normalisation for Polaris to be done either using relative (`pdf`) or absolute normalisation (`pdf_norm`). This manifests as the ability to set `mode : pdf` or `mode : pdf_norm` within your `polaris_config.yaml`. Alternatively this can be done within scripts using `setup_inst_object(mode="PDF")` or `setup_inst_object(mode="PDF_NORM")` and equivalently elsewhere. Some changes have also happened under the hood to remove where relative normalisation was being forced.
```

Comment:
How about something like this that includes the link to the POLARIS docs?
```suggestion
-  Renamed parameter ``mode="pdf_norm"`` to :ref:`ISIS POLARIS powder reduction<isis-powder-diffraction-polaris-ref>` for when the chopper is off/stationary with a default ``van_normalisation_method="absolute"``. The existing  ``mode="pdf"`` now has a default  ``van_normalisation_method="relative"`` (was previously ``"relative"``).
```
I think linking to the docs means we can ditch the stuff about how the user can change the setting (helpful though that was).
What do you think?